kdc/kerberos5.c

```diff
@@ -2032,6 +2032,59 @@ add_enc_pa_rep(astgs_request_t r)
 			   KRB5_PADATA_FX_FAST, NULL, 0);
 }
 
+static krb5_error_code
+add_supported_enctypes(astgs_request_t r)
```

Comment:
Please put a comment referencing [MS-KILE] 2.2.7 so we know that this is a MS extension to Kerberos.

Reply:
Ideally we would put this into the backend given it's very Windows-specific... but, I guess, we don't want to expose `astgs_request_desc` to the plugin API...